include/fizzy/fizzy.h

```diff
@@ -133,6 +133,33 @@ typedef struct FizzyExternalGlobal
     FizzyGlobalType type;
 } FizzyExternalGlobal;
 
+/// External kind.
+typedef enum FizzyExternalKind
+{
+    FizzyExternalKindFunction,
+    FizzyExternalKindTable,
+    FizzyExternalKindMemory,
+    FizzyExternalKindGlobal
+} FizzyExternalKind;
+
+/// Import description.
+///
+/// @note Only one member of FizzyImportDescription::desc union corresponding to
+/// FizzyImportDescription::kind is defined for each import.
+typedef struct FizzyImportDescription
```

Comment:
Do we need to mention the lifetime of module/name here? It is explained in `fizzy_get_import_description`.

Perhaps what we could note here is that the user should refer to the documentation from the function returning this value about lifetimes:
```suggestion
typedef struct FizzyImportDescription
//// For the lifetime of the #module and #kind fields, please refer to the description provided by the function used to obtain this structure.
```

Is `#module` the correct way?

Reply:
Yes, `#module` and `#name` worked, but I needed to add individual documentation to the members.